packages/mux-player/src/media-theme-mux/media-theme-mux.ts

```diff
@@ -153,13 +151,15 @@ export const VodChromeSmall = (props: ThemeMuxTemplateProps) => html`
     ${MediaPlayButton()}
     ${MediaSeekForwardButton(props)}
   </div>
+  <media-time-range></media-time-range>
   <media-control-bar>
-    <media-time-range></media-time-range>
     <mxp-time-display></mxp-time-display>
     ${MediaMuteButton()}
     <media-volume-range></media-volume-range>
+    <div class="mxp-spacer"></div>
     <media-playback-rate-button></media-playback-rate-button>
     ${MediaFullscreenButton()}
+    <div class="mxp-padding-2"></div>
```

Comment:
non-blocking: assuming this was a path of less resistance for things like background compared to e.g. just adding padding to the media-control-bar?